docs/latest/concepts/forms.md

```diff
@@ -12,11 +12,12 @@ In the browser, a `<form>` submit will send an HTML action (usually `GET` or
 
 ## POST request with `multipart/form-data`
```

Comment:
Nit: We should probably update the heading too

Reply:
Fixed!